src/test/java/LoginTests.java

```diff
@@ -21,7 +21,7 @@ public static void LoginEmptyEmailPasswordTest () {
         driver.quit();
     }
     @Test
-    public static void RegistrationTest(){
+    public static void RegistrationNavigation(){
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example  `public static void registrationNavigation()`

![image](https://user-images.githubusercontent.com/104761482/215409375-fb6c8ce2-9392-449b-a55c-50e439377b31.png)